### ISSUE TITLE
Support more systems on the PPA

### ISSUE THREAD

**Issue by cobalt2727** [MrGlockenspiel/activate-linux#184]
Could you try messing with the package settings to support more architectures and 18.04 if possible?

**Comment by MrGlockenspiel** [comment]
@eddelbuettel runs the PPA, we'd have to take it up with them

**Comment by MrGlockenspiel** [comment]
id like to get into the main repos of debian/ubuntu but because of the previous license we couldnt before and also i have no idea how to do any of that

**Comment by eddelbuettel** [comment]
@MrGlockenspiel  I am actually a card-carrying Debian dev (since 1995) and _could_ upload and adopt but sometimes ... that is work and I frankly unaware of whatever standards we use for desktop apps (or whatever categories this falls into). (I also have no sway over ftpmasters: if they find something amiss in terms of licensing there is little you can do apart.  Apart from changing and complying which, generally speaking and absent of specifics, is the sensible way.)

Simple PPAs are so much easier...

**Comment by cobalt2727** [comment]
@eddelbuettel 

18.04 support (ideally, if all goes well without a build failure) can be added by opening https://launchpad.net/~edd/+archive/ubuntu/misc/+copy-packages while you're logged in and copying the packages to additional `Destination series`

Adding support for more architectures is _way_ easier, just check some extra boxes on https://launchpad.net/~edd/+archive/ubuntu/misc/+edit

**Comment by eddelbuettel** [comment]
@cobalt2727 With all due respect:   I said I am a Debian maintainer; you ask about a (seriously outdated) Ubuntu release.  Not the same thing.  

If you have an itch for different things at Launchpad, _just apply to Launchpad and create your PPA_ as you evidently do not even have to do anything to the `activate-linux` sources: just take'em from my PPA. 

For me this was a mostly one-off endeavor once: I set the the package, I updated it once, I ran it twice and likely tweeted twice.  I don't really have all that much bandwidth for `activate-linux`, and for what it is worth I run 22.10 on my server (likely going to 23.04 very soon) and 23.04 on a laptop.  So that would be a polite "no I won't" from me.
